testing/TestHarness/TestHarness/TestHarness.csproj

```diff
@@ -76,17 +76,19 @@
 		<ProjectReference Include="..\TestHarness.Core\TestHarness.Core.csproj" />
 
 		<PackageReference Include="Uno.Core.Extensions.Logging.Singleton"/>
+		<PackageReference Include="Uno.WinUI" />
+		<PackageReference Include="Uno.UI.Adapter.Microsoft.Extensions.Logging"/>
```

Comment:
Shouldn't these be added implicitly by Uno.Sdk?

Reply:
They need to be explicit in order for the canary updater to work as it was complaining about version mismatch